packages/seacas/libraries/ioss/src/Ioss_DatabaseIO.h

```diff
@@ -356,7 +358,13 @@ namespace Ioss {
     void get_step_times()
     {
       IOSS_FUNC_ENTER(m_);
-      return get_step_times_nl();
+      get_step_times_nl();
+    }
+
+    std::vector<double> get_db_step_times()
```

Comment:
I know you don't like comments, but how does a client know whether to use this function or the `get_step_times()` function?

Reply:
I'll add the necessary comments